.changeset/big-emus-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+"wait-for-workflow-job": patch
```

Comment:
maybe a `minor` as an init?

Reply:
Fixed in: https://github.com/smartcontractkit/.github/pull/454/commits/9adb0ad6edebf171b0e9180215afad8d0835dabb